ansible/roles/providers/tasks/azure-redhat.yml

```diff
@@ -18,35 +18,22 @@
 - block:
     # TODO: remove cut utility fix to initramfs after its been fixed upstream
     # https://bugzilla.redhat.com/show_bug.cgi?id=2165042
-    - name: expose cut utility to initramfs setup
+    - name: Expose cut utility to initramfs setup
       shell: echo 'install_items+=" /usr/bin/cut "' > /etc/dracut.conf.d/cut.conf
 
-    - name: refresh dracut
+    - name: Refresh dracut
       shell: dracut -f
 
-    # https://d2iq.atlassian.net/browse/D2IQ-96417 removes extra 4 and 5 partitions so that growpart cloud-init module can expand the root filesystem to the rest of the disk.
-    #
-    # [packer@pkrvmia9ljyrxtg ~]$ lsblk -oMOUNTPOINT,PKNAME -P | grep 'MOUNTPOINT="/"' | cut -d= -f3  | xargs
-    # sda
-    - name: Get Device that holds root filesystem
-      shell: lsblk -oMOUNTPOINT,PKNAME -P | grep 'MOUNTPOINT="/"' | cut -d= -f3  | xargs
-      register: rootdevice
-
-    - name: Loop over ansible block devices
-      parted:
-        number: "{{ item }}"
-        state: absent
-        device: "/dev/{{ rootdevice.stdout }}"
-      with_items:
-        - 4
-        - 5
-
-    - name: unconditionally reboot the machine with all defaults
-      ansible.builtin.reboot:
-  when:
-    - packer["image_publisher"] == "erockyenterprisesoftwarefoundationinc1653071250513" and packer["image_version"] == "9.1.20230215"
```

Comment:
Don't need this workaround anymore with 9.3 image as I see last partition is meant for root so it grow if needed -
```
[root@manoj-rocky-9 ~]# lsblk
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINTS
sda      8:0    0   10G  0 disk
├─sda1   8:1    0   99M  0 part /boot/efi
├─sda2   8:2    0 1000M  0 part /boot
├─sda3   8:3    0    4M  0 part
├─sda4   8:4    0    1M  0 part
└─sda5   8:5    0  8.9G  0 part /
sdb      8:16   0   16G  0 disk
└─sdb1   8:17   0   16G  0 part /mnt
sr0     11:0    1  628K  0 rom

[root@manoj-rocky-9 ~]# cat /etc/os-release
NAME="Rocky Linux"
VERSION="9.3 (Blue Onyx)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.3"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Rocky Linux 9.3 (Blue Onyx)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:9::baseos"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2032-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-9"
ROCKY_SUPPORT_PRODUCT_VERSION="9.3"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.3"
```